src/ngrx/7-create-api-effects/create-api-effects.md

```diff
@@ -34,29 +36,50 @@ TODO
 
 </details>
 
-## P1: Solution
+## P1: What You Need to Know
 
-<details>
-<summary>src/app/store/login/login.effects.ts</summary>
+[NgRx Effects](https://ngrx.io/guide/effects) is a side-effect model that utilizes RxJS to react to Actions being dispatched to manage side-effects such as network requests, web socket messages and time-based events. One thing that Effects are not responsible for is updating state; this is a responsiblity for Reducers ([learn-ngrx/create-reducer]).
```

Comment:
Should this be "are a side-effect model"? Grammar is hard

Reply:
I was thinking of this too. I'm basically taking from this quote:

```
Effects are an RxJS powered side effect model for [Store](https://ngrx.io/guide/store). Effects use streams to provide [new sources](https://martinfowler.com/eaaDev/EventSourcing.html) of actions to reduce state based on external interactions such as network requests, web socket messages and time-based events.
```

https://ngrx.io/guide/effects

So yea, after reading again, I think it should be "are" and not "is"